subsys/dfu/dfu_target/Kconfig

```diff
@@ -117,14 +117,51 @@ config DFU_TARGET_SUIT
 	bool "SUIT DFU target support"
 	default y
 	depends on SUIT
+	depends on SDFW_SERVICES_ENABLED
 	imply MPU_ALLOW_FLASH_WRITE
 	select SUIT_UTILS
 	select ZCBOR
 	select ZCBOR_CANONICAL
+	select FLASH
+	select STREAM_FLASH
+	select STREAM_FLASH_ERASE
 	select DFU_TARGET_STREAM
+	# Minimal SUIT configuration to support SUIT Cache processing
```

Comment:
Does it mean there is no real use case to use `DFU_TARGET_SUIT` without cache processing?
If I correctly recall, `select`, if used, is for hard requirements (like `depends on`).
If optional stuff is to be included, the `imply` is a more suitable choice.